_posts/en/posts/2015-12-23-capacity-increases-faq.md

```diff
@@ -76,6 +76,16 @@ Segwit does require more changes in higher level software stacks than a simple b
 
 Developers, miners, and the community have accrued significant experience deploying soft forks, and we believe segwit can be deployed at least as fast, and probably more securely, than a hard fork that increases the maximum block size.
 
+## I read that a hard fork deployment of segregated witness would be preferable to avoid complexity, is that correct? {why-not-hardfork}
+
+Segregated witness, like CHECKSEQUENCEVERIFY of BIP 68 & 112, was first prototyped in the Elements Alpha sidechain. Like CSV, the implementation that finally made it into Bitcoin Core was different from the initial prototype, for various reasons:
+
+1. Alpha was a prototype chain, and there was a lot that learned from using it in production, even on just a test network. The Alpha version of SegWit was a "just don't include the signatures, etc., in the hash" hard-fork change. With the experience of using this code on a testnet sidechain, and performing third-party integrations, it was discovered that this approach has significant drawbacks: it is an inefficient use of block space size; it requires messy, not-obviously-correct code in the core data structures of bitcoin; and it totally and completely breaks all existing infrastructure in weird, unexpected, layer-violating, and unique ways. The tweak introduced by Bitcoin Core developer Luke-Jr to allow SegWit to be soft-fork compatible also fixes all these issues. It's an objectively better approach regardless of hard-fork vs soft-fork, for code engineering reasons. 
+
+2. The idea itself was refined and improved over time as new insights were had. Luke-Jr's approach made script versioning very easy (1 byte per output) to add. Script versioning allows us to fix all sorts of long-standing problems with the bitcoin scripting language. To ease review, the first SegWit script version only makes absolutely uncontroversial fixes to security problems like quadratic hashing, but much more (like aggregate Schnorr signatures) becomes possible. So today's SegWit is different from and better than earlier proposals because it has received more care and attention from its creators in the elapsed time.
```

Comment:
s/So today's SegWit is different from and better than earlier proposals/Today's SegWit has benefited from the lessons learned from previous implementations and the cumulative review of dozens of contributors since the first iteration, significantly reducing technical debt and increasing extensibility/